locales/index.js

```diff
@@ -0,0 +1,11 @@
+function getLocales(arr) {
+  return arr.map(function(item) {
+    return item.replace("./", "").replace(".json", "");
+  });
+}
+
+var reqF = require('enhanced-require');
+var req = reqF(module);
```

Comment:
Not sure why I didn't thought of this lol! It can pretty much run both client and server with this module.